### PR TITLE
Add app.fotema.Fotema

### PR DESCRIPTION
**Pull request by blissd**
Fotema is a photo gallery app.

### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/blissd/fotema
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by blissd** [comment]
Permission `--filesystem=xdg-pictures:ro` is to provide access to the user's pictures directory so the Fotema can display those pictures.

File: app.fotema.Fotema.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "add-ld-path": "lib",
+      "bundle": true,
+      "autodelete": true
+    }
+  },
+  "modules": [
+    "modules/libde265.json",
+    "modules/libheif.json",
+    "modules/glycin-loaders.json",
+    "modules/ffmpeg.json",
```
Comment (bbhtt) [inline app.fotema.Fotema.json:46]: Please add these to the main manifest here.

File: app.fotema.Fotema.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+  "id": "app.fotema.Fotema",
```
Comment (bbhtt) [inline app.fotema.Fotema.json:2]: I don't see an webpage at fotema.app Is the domain yours? How can I verify?

Comment (blissd) [reply app.fotema.Fotema.json:2]: I bought the domain, but have only setup email. Feel free to email hello@fotema.app and I will reply.

Comment (bbhtt) [reply app.fotema.Fotema.json:2]: Preferably, just setting up a blank page and linking https://github.com/blissd/fotema will be enough.

If you can't do that please send an email to `admins@flathub.org`, confirming the submission.

Comment (blissd) [reply app.fotema.Fotema.json:2]: https://fotema.app/ now exists and points to the Fotema github repo.

File: app.fotema.Fotema.json (hunk 1 above)
Comment (blissd) [reply app.fotema.Fotema.json]: Done.

Comment (bbhtt) [reply app.fotema.Fotema.json]: You can but keeping them in the main manifest is better.

File: app.fotema.Fotema.json (hunk 2 above)
Comment (bbhtt) [reply app.fotema.Fotema.json:2]: You have a HSTS header set I can't open the page

Comment (blissd) [reply app.fotema.Fotema.json:2]: Are you using http instead of https?

Comment (blissd) [reply app.fotema.Fotema.json:2]: I think HSTS is mandatory for .app domains.

File: app.fotema.Fotema.json (hunk 1 above)
Comment (hfiguiere) [reply app.fotema.Fotema.json]: you shoudln't. it makes it hard to maintain. and when a maintainer goes MIA it team member like @bbhtt and I that pick up the pieces.

File: app.fotema.Fotema.json (hunk 2 above)
Comment (bbhtt) [reply app.fotema.Fotema.json:2]: I can see the full webpage now, maybe the changes you were making were still propagating. 

Thanks.

**Comment by bbhtt** [comment]
bot, build app.fotema.Fotema

File: app.fotema.Fotema.json [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  ],
+  "command": "fotema",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures:ro",
+    "--env=RUST_BACKTRACE=1"
```
Comment (bbhtt) [inline app.fotema.Fotema.json:18]: Do you need the logs? Seems it is better to turn this off to avoid log spam.

**Comment by bbhtt** [comment]
Seems ok to me. Please test the build.

File: app.fotema.Fotema.json (hunk 3 above)
Comment (blissd) [reply app.fotema.Fotema.json:18]: Hmm... I agree that the 1.0.0 release is logging too much. However, to fix that properly I think I need to change some of the logging configuration in the code and cut a new release.

**Comment by blissd** [comment]
> Build [120301 successful](https://buildbot.flathub.org/#/builders/6/builds/120301) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/103162/app.fotema.Fotema.flatpakref
> ```

I have installed the build release and it appears to work.

File: app.fotema.Fotema.json (hunk 3 above)
Comment (bbhtt) [reply app.fotema.Fotema.json:18]: This is a minor issue, you can fix this post merge.

**Pull request closed without merging**